bot/interact.go

```diff
@@ -128,6 +112,49 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 	}
 }
 
+// Runs validations over the submitted salary range and job offer link. Produces a list of errors if any.
```

Comment:
usually the first word is the type you are commenting (otherwise go vet will complain)
```suggestion
// ValidateSubmission runs validations over the submitted salary range and job offer link. Produces a list of errors if any.
```